partiql-plan/src/main/kotlin/org/partiql/plan/v1/rel/Rel.kt

```diff
@@ -0,0 +1,15 @@
+package org.partiql.plan.v1.rel
+
+import org.partiql.plan.v1.Node
+import org.partiql.plan.v1.Schema
+
+public interface Rel : Node {
+
+    public fun getInputs(): List<Rel>
```

Comment:
So most of the `Rel`s have a `getInput(): Rel` function. Should they actually just use this `getInputs`s function?

Reply:
Some of them have getInput() when it's a single input, and some have getLeft() and getRight() if it's a multi-set operator. This is a top-level getInputs() for tree-traversal